src/server/api/controllers/products.controller.js

```diff
@@ -1,11 +1,27 @@
 const knex = require('../../config/db');
-// const HttpError = require('../lib/utils/http-error');
+const HttpError = require('../lib/utils/http-error');
 // const moment = require('moment-timezone');
 
 const getProducts = async () => {
   return knex('products');
 };
 
+const getProductById = async (id) => {
+  if (!id) {
+    throw new HttpError('Id should be a number', 400);
+  }
+
+  try {
+    const products = await knex('products').select('products.*').where({ id });
```

Comment:
Hey Dyvia .... it looks okay just a little detai here can you make for the products by id to return just the following (id,name,price,color,size,is_available,stock_amount,is_on_discount,discount_percent,picture) like mentioned in the issue#86